client/test/cypress/integration/admin/publications.spec.js

```diff
@@ -25,36 +26,34 @@ describe('Admin Publications', () => {
     cy.dataCy('new_publication_input')
       .type('{enter}');
     cy.dataCy('publications_list');
-    cy.dataCy('create_publication_notify').should('be.visible').should('have.class','bg-negative');
-    cy.injectAxe();
+    cy.dataCy('name_field_error').should('be.visible');
     cy.dataCy('new_publication_input')
       .type('Draft Publication from Cypress');
     cy.checkA11y();
   });
 
   it('prevents publication creation when the name exceeds the maximum length', () => {
-    const name_256_characters = '0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345';
+    const name_256_characters = '01234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123450{enter}';
     cy.dataCy('new_publication_input')
       .type(name_256_characters + '{enter}');
-    cy.dataCy('banner_form_error').should('be.visible').should('have.class','bg-negative');
-    cy.injectAxe();
+    cy.dataCy('name_field_error').should('be.visible');
+    cy.get('.q-transition--field-message-leave-active').should('not.exist');
     cy.checkA11y();
   });
 
   it('prevents publication creation when the name is not unique', () => {
     cy.dataCy('new_publication_input')
       .type('Duplicate Publication from Cypress{enter}');
-    cy.dataCy('create_publication_notify').should('be.visible').should('have.class','bg-positive');
-    cy.injectAxe();
+    cy.dataCy('create_publication_notify').should('be.visible');
```

Comment:
Wouldn't it be a stronger test to assert the type of the notification that's expected in addition to the notification being visible?
```suggestion
    cy.dataCy('create_publication_notify').should('be.visible').should('have.class','bg-positive');
```

Reply:
You're right, but I had a reason why I changed it.  I'll investigate why I did that and update.